main/acle.md

```diff
@@ -9456,6 +9461,701 @@ are named after. All of the functions have external linkage.
+   // _u32_x4, _s32_x4 and _f32_x4
+   void svst1[_u32_x2](svcount_t png, uint32_t *rn, svuint32x2_t zt);
+   void svst1_vnum[_u32_x2](svcount_t png, uint32_t *rn, int64_t vnum,
+                            svuint32x2_t zt);
+
+
+   // Variants are also available for _s64_x2, _f64_x2
+   void svst1[_u64_x2](svcount_t png, uint64_t *rn, svuint64x2_t zt);
+   void svst1_vnum[_u64_x2](svcount_t png, uint64_t *rn, int64_t vnum,
+                            svuint64x2_t zt);
+
+
+   // Variants are also available for _s64_x4, _f64_x4
+   void svst1[_u64_x4](svcount_t png, uint64_t *rn, svuint64x4_t zt);
+   void svst1_vnum[_u64_x4](svcount_t png, uint64_t *rn, int64_t vnum,
+                            svuint64x4_t zt);
+
+   ```
+
+#### STNT1B, STNT1D, STNT1H, STNT1W
+
+Contiguous non-temporal store of multi-vector operand.
+
+ ``` c
+   // All the intrinsics below are: [SVE2.1 or SME2]
+   // Variants are also available for _s8_x2
+   void svstnt1[_u8_x2](svcount_t png, uint8_t *rn, svuint8x2_t zt);
+   void svstnt1_vnum[_u8_x2](svcount_t png, uint8_t *rn, int64_t vnum,
+                             svuint8x2_t zt);
+
+   // Variants are also available for _s8_x4
+   void svstnt1[_u8_x4](svcount_t png, uint8_t *rn, svuint8x4_t zt);
+   void svstnt1_vnum[_u8_x4](svcount_t png, uint8_t *rn, int64_t vnum,
+                             svuint8x4_t zt);
+
+
+   // Variants are also available for _s16_x2, _f16_x2 and _bf16_x2
+   void svstnt1[_u16_x2](svcount_t png, uint16_t *rn, svuint16x2_t zt);
+   void svstnt1_vnum[_u16_x2](svcount_t png, uint16_t *rn, int64_t vnum,
+                              svuint16x2_t zt);
+
+   // Variants are also available for _s16_x4, _f16_x4 and _bf16_x4
+   void svstnt1[_u16_x4](svcount_t png, uint16_t *rn, svuint16x4_t zt);
+   void svstnt1_vnum[_u16_x4](svcount_t png, uint16_t *rn, int64_t vnum,
+                              svuint16x4_t zt);
+
+
+   // Variants are also available for _s32_x2 and _f32_x2
+   void svstnt1[_u32_x2](svcount_t png, uint32_t *rn, svuint32x2_t zt);
+   void svstnt1_vnum[_u32_x2](svcount_t png, uint32_t *rn, int64_t vnum,
+                              svuint32x2_t zt);
+
+   // Variants are also available for _s32_x4 and _f32_x4
+   void svstnt1[_u32_x4](svcount_t png, uint32_t *rn, svuint32x4_t zt);
+   void svstnt1_vnum[_u32_x4](svcount_t png, uint32_t *rn, int64_t vnum,
+                              svuint32x4_t zt);
+
+
+   // Variants are also available for _s64_x2 and _f64_x2
+   void svstnt1[_u64_x2](svcount_t png, uint64_t *rn, svuint64x2_t zt);
+   void svstnt1_vnum[_u64_x2](svcount_t png, uint64_t *rn, int64_t vnum,
+                              svuint64x2_t zt);
+
+   // Variants are also available for _s64_x4 and _f64_x4
+   void svstnt1[_u64_x4](svcount_t png, uint64_t *rn, svuint64x4_t zt);
+   void svstnt1_vnum[_u64_x4](svcount_t png, uint64_t *rn, int64_t vnum,
+                              svuint64x4_t zt);
+   ```
+
+#### ST1Q
+
+Scatter store quadwords.
```

Comment:
There is only an unscaled version of this instruction? So maybe don't have both index and offset?